Checker/compression_test.fs

```diff
@@ -31,6 +31,8 @@ let testFiles = [
     "buoy.frag"
     "orchard.frag"
     "robin.frag"
+    "sult.frag"
+    "../unit/loop.frag"
```

Comment:
I'd rather avoid synthetic tests in this list.

Reply:
Me too. But I wanted to know if this rewrite also reduces compressed size. We need to grab more real test shaders.